utils/scripts/create_parameter_json_from_nfcore_schema.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/env python3
+
+"""
+Generates a parameter json for AWS HealthOmics workflows
+from NF-CORE nextflow schema JSON.
+"""
+
+from argparse import ArgumentParser
+import json
+from pprint import pprint
+import logging
+
+# Create a logger
+logging.basicConfig(format="%(name)s - %(asctime)s %(levelname)s: %(message)s")
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument('--nfcore_schema_file', type=str, required=True,
+                    help="nextflow schema JSON used in NF-CORE nextflow workflows")
+parser.add_argument('--output_file', type=str, required=True,
+                    help="output file name")
+    
+def create_param_json(schema):
+    parameter_json_object = {}
+    
+    if "definitions" not in schema:
+        raise KeyError("Schema does not contain key: definitions")
+    
+    for key, value in schema["definitions"].items():
+        logger.info("processing key: %s" % key)
+        if "required" in value:
+            required_params = value["required"]
+        else:
+            required_params = []
```

Comment:
This doesn't need to be in the loop. It has a risk of emptying out collected required parameters.